saleor/account/models.py

```diff
@@ -218,8 +218,11 @@ def has_perms(self, perm_list):
         if not self.is_active:
             return False
 
-        wanted_perms = {perm.value for perm in perm_list}
-        actual_perms = self._get_permissions()
+        try:
+            wanted_perms = {perm.value for perm in perm_list}
+        except AttributeError:
+            wanted_perms = set(perm_list)
```

Comment:
It's a workaround (similar in `has_perm` method):  `has_perms` and `has_perm` shouldn't work like that. Here is the issue for that. https://github.com/mirumee/saleor/issues/5527